### DISCUSSION TITLE
Hover Text for Tabs

### DISCUSSION THREAD

**Discussion by LittleMonkeyMojo** [sumatrapdfreader/sumatrapdf#2658] [Ideas]
I, quite frequently, open more than a few PDFs at once.  If I open more than a couple, the tab text noting the filename is truncated, and with the way many of these PDFs are named the tab text noting the filename is truncated even if it's the only PDF open.

Would it be possible to add the ability to hover and have pop-up text (what's it called, a "balloon") with the full name of the doc being hovered over?

Just a thought.

**Comment by kjk** [comment]
Starting with 3.4 we show the full path of the file when hovering over tabs. You can upgrade to the latest version: https://www.sumatrapdfreader.org/download-free-pdf-viewer

**Comment by GitHubRulesOK** [comment]
@LittleMonkeyMojo 
>tab text noting the filename is truncated even if it's the only PDF open.

With the balloon/tooltips showing long name you can use short names in wider tabs

```
FullPathInTitle = false
TabWidth = 300
```
They will automatically get smaller as and when required
